coreneuron/io/nrn_soa.hpp

```diff
@@ -0,0 +1,36 @@
+#pragma once
```

Comment:
nitpick : I would rename this file to something like `mem_layout.hpp` or `mem_layout_util.hpp`

Reply:
Done